wger/exercises/templates/exercise/overview.html

```diff
@@ -8,7 +8,22 @@
 <!--
         Title
 -->
-{% block title %}{% trans "Exercises" %}{% endblock %}
+{% block title %}
+{% trans "Exercises" %}
+<div class="btn-group" style="float:right;">
+    <button type="button" class="btn btn-primary dropdown-toggle btn-xs" data-toggle="dropdown">
+            {% trans "Filter exercises by language" %}  <span class="caret"></span>
```

Comment:
We can update this button to less words as: `Filter by language`

Reply:
Yeah, good.